php-nginx/build-scripts/build_php70.sh

```diff
@@ -49,6 +49,9 @@ rm mailparse.tar.gz
 # rm apcu.tar.gz
 git clone https://github.com/krakjoe/apcu.git ${PHP_SRC}/ext/apcu
 
+# APC compatibility layer
+git clone https://github.com/krakjoe/apcu-bc.git ${PHP_SRC}/ext/apc
```

Comment:
That would be awesome, but it's up to you.

Also can you add apc to the test?
https://github.com/GoogleCloudPlatform/php-docker/blob/master/tests/PHP7ExtensionsTest.php